www_src/pages/section-2/page.jsx

```diff
@@ -0,0 +1,19 @@
+var React = require('react');
+
+module.exports = React.createClass({
+  render: function () {
+    var style = {};
+
+    if (this.props.screenshot !== 'EMPTY') {
```

Comment:
this seems like a "magic" string - it's probably better to not rely on a special value but instead simply tap into "was there a screenshot at all passed in". If so, style, if not, no style?